vnext/PropertySheets/React.Cpp.props

```diff
@@ -50,9 +50,13 @@
   </ItemDefinitionGroup>
 
   <PropertyGroup>
+    <!-- This prop is used for shared JSI code and headers -->
     <JSI_Source Condition="'$(JSI_Source)' == ''">$(ReactNativeDir)\ReactCommon\jsi</JSI_Source>
     <JSI_Source Condition="'$(USE_HERMES)' == 'true'">$(HERMES_Package)\installed\x64-windows\include\jsi_ref</JSI_Source>
-    <JSI_Source Condition="'$(USE_V8)' == 'true'">$(V8_Package)\installed\x64-windows\include\jsi_ref</JSI_Source>
+
+    <!-- This prop is used for JSI code that may be overridden in external JSI implementations -->
```

Comment:
>JSI code that may be overridden in external JSI implementations [](start = 31, length = 63)

Which files are we overriding in $(V8_Package)\build\native\include?

Reply:
jsi\instrumentation.h
jsi\jsi.cpp
jsi\jsi.h
jsi\jsi-inl.h
ScriptStore.h

---
In reply to: [387968283](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387968283) [](ancestors = 387968283)